### PR TITLE
lingering org transition cleanup

### PR DESCRIPTION
**Pull request by tequilarista**
Deleted old non-setup-gcloud actions code that's since been migrated to their own repos within the new org.  Cleaned out superfluous workflows related to same.

Uplifted setup-cloud/setup-cloud/* to setup-gcloud/.

Uplifted lint config files to top level and cleaned up changes in setupGcloudSDK related to this.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -13,10 +13,10 @@
 # limitations under the License.
 
 name: Set up gcloud Cloud SDK environment
-author: Google LLC
+author: GoogleCloudPlatform
```
Comment (averikitsch) [inline action.yml:16]: Changes to this file need to be reverted (this is for the deprecate code)

File: setup-gcloud/README.md [hunk 2]
```diff
@@ -25,74 +25,3 @@ steps:
 +  uses: google-github-actions/setup-gcloud@master
```
Comment (averikitsch) [inline setup-gcloud/README.md:25]: I think we can go ahead and delete this file. @bharathkkb do you think we should add a note in the readme about the deprecated actions?

Comment (bharathkkb) [reply setup-gcloud/README.md:25]: Yeah I think we should add a readme note stating these are not deprecated, but rather live elsewhere.

File: action.yml [hunk 3]
```diff
@@ -70,4 +69,4 @@ inputs:
 
 runs:
   using: node12
-  main: setup-gcloud/dist/index.js
+  main: dist-frozen/index.js
```
Comment (bharathkkb) [inline action.yml:72]: we can remove `dist-frozen`. this should point to `dist/index.js`

File: action.yml (hunk 1 above)
Comment (tequilarista) [reply action.yml]: reverted.

File: action.yml (hunk 3 above)
Comment (tequilarista) [reply action.yml]: done

File: .github/workflows/setup-gcloud-it.yml [hunk 4]
```diff
@@ -3,7 +3,7 @@ name: setup-gcloud Integration
 on:
   push:
     paths:
-    - 'setup-gcloud/**'
+    - './**'
     - '.github/workflows/setup-gcloud*'
```
Comment (bharathkkb) [inline .github/workflows/setup-gcloud-it.yml:7]: nit: Now that we have top leveled this, we can use
```yaml
  push:
    branches:
    - 'main'
  pull_request:
```
for parity with other actions test workflows
https://github.com/google-github-actions/deploy-cloudrun/blob/c8ab46dee1f5dad5d4ec6e92519bf90fa9318923/.github/workflows/deploy-cloudrun-it.yml#L3

File: setup-gcloud/README.md (hunk 2 above)
Comment (bharathkkb) [reply setup-gcloud/README.md:25]: @tequilarista I am thinking we can move the deprecation notice to main root readme for increased visibility and get rid of `setup-gcloud` dir completely?

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Some minor changes but looks good overall. Thanks for taking care of this!

- There seems to be a diff for`setup-gcloud/dist/index.js → dist/index.js`. It maybe because it was built locally and added in a commit. We should remove this checked in file and just rename it without changing source code, our build workflow should generate a new version of this.
- We should update [build/release workflow](https://github.com/google-github-actions/setup-gcloud/blob/be078c0cd4467957aec54a8ac035fbe9bbdc0a0c/.github/workflows/release-please.yml#L19) as`setup-gcloud` dir no longer exists. I propose we just  build/release setup-gcloud as @averikitsch is splitting out the SDK anyway.

File: tsconfig.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Google LLC
+ * Copyright 2019 Google LLC
```
Comment (iennae) [inline tsconfig.json:2]: why this change? seems weird.

Comment (tequilarista) [reply tsconfig.json:2]: came over with one of the files moves, I can put it back

File: setup-gcloud/README.md (hunk 2 above)
Comment (tequilarista) [reply setup-gcloud/README.md:25]: makes sense, will update

**Review by bharathkkb — APPROVED** [review]
LGTM - did a quick test [here](https://github.com/bharathkkb/test-setup-gcloud/blob/main/.github/workflows/gcloud.yml) for setup-gcloud

The dist changes are there but I think we can just let it get recreated after merging this PR.

/cc @averikitsch for another 👀

**Review by averikitsch — APPROVED** [review]
